src/connectors/indie88.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+// Indie88 has the track and artist flipped on their main web player. The values are intentionally flipped in this Connector
+
+Connector.playerSelector = [
```

Comment:
The `playerSelector` property supports strings only.

Ideally, I would split the connector; for now you could use "setup" pattern (for example, https://github.com/web-scrobbler/web-scrobbler/blob/master/src/connectors/douban-artists.js).